app/schema_validation/__init__.py

```diff
@@ -58,6 +59,32 @@ def validate_schema_date_with_hour(instance):
     return True
 
 
+@format_checker.checks('send_a_file_retention_period', raises=ValidationError)
+def validate_schema_retention_period(instance):
+    if instance is None:
+        return True
+
+    if isinstance(instance, str):
+        period = instance.strip().lower()
+        match = re.match(r'^(\d+) weeks?$', period)
+        if match and 1 <= int(match.group(1)) <= 78:
+            return True
+
+    raise ValidationError(
+        f"Unsupported value for retention_period: {instance}. Supported periods are from 1 to 78 weeks."
```

Comment:
bit late to the party now that we've already merged a bunch of docs changes, soz. one for @karlchillmaid but i'm not 100% sold on this error message as a general case.

i think when a user inputs `"99 weeks"` this is a good error message to return. 

However, I don't think it (or the API docs) are explicit enough on how to fix the input if a user enters other incorrect values.

things a user might foreseeably and reasonably try the following and expect them to work:

* `4` (the number, not a string)
* `"4"` (as a string but with no date unit)
* `"28 days"` or `"1 month"` (a string but the wrong date unit)

i guess i'm worried if a user enters one of those, we're telling them what the supported _date range_ is but not what the supported _format_ is. Thoughts?

Reply:
I broadly agree - I suggested one that I thought indicated the input values more clearly but didn't push back when Karl suggested alternatives here: https://github.com/alphagov/notifications-tech-docs/pull/172#discussion_r958611629

Reply:
Maybe we can take these messages and iterate if we hear from users that they're not clear enough?

Reply:
Cool, lets agree to look at logs in a week or two and see what people have been trying to send in 👍